pkg/arvo/app/claz.hoon

```diff
@@ -484,6 +493,8 @@
     %transfer-ship  (transfer-ship:dat +.call)
     %set-transfer-proxy  (set-transfer-proxy:dat +.call)
     %adopt  (adopt:dat +.call)
+  ::
```

Comment:
Why the extra `::`?  Is it because it's not an azimuth-native call?

Reply:
Yeah, because it goes to a contract different than the Ecliptic. Note that we separate it out [in the type definition](https://github.com/urbit/urbit/pull/1809/files#diff-bd1093e2e574caa62df0552afecc07fcR85-R86) as well.